gnomad/utils/constraint.py

```diff
@@ -125,3 +135,246 @@ def collapse_strand(
         if isinstance(t, hl.Table)
         else t.annotate_rows(**collapse_expr)
     )
+
+
+def build_models(
+    coverage_ht: hl.Table,
+    weighted: bool = False,
+    pops: Tuple[str] = (),
+    keys: Tuple[str] = (
+        "context",
+        "ref",
+        "alt",
+        "methylation_level",
+        "mu_snp",
+    ),
+    cov_cutoff: int = COVERAGE_CUTOFF,
+) -> Tuple[Tuple[float, float], hl.StructExpression]:
+    """
```

Comment:
I reformat the docstring a little bit. I didn't delete some sentence because it make more sense to keep it under this format